doc/classes/XROrigin3D.xml

```diff
@@ -5,20 +5,19 @@
 	</brief_description>
 	<description>
 		This is a special node within the AR/VR system that maps the physical location of the center of our tracking space to the virtual location within our game world.
-		There should be only one of these nodes in your scene and you must have one. All the XRCamera3D, XRController3D and XRAnchor3D nodes should be direct children of this node for spatial tracking to work correctly.
+		There must be only one of these nodes in the scene tree at the time. All the [XRCamera3D], [XRController3D], and [XRAnchor3D] nodes should be direct children of this node for spatial tracking to work correctly.
```

Comment:
This isn't exactly right. Similar to `Camera3D`, there should only be one `XROrigin3D` that has `current` set to `true`, but multiple can exist in the scene tree at once.

One pattern is to load a new scene with its own `XROrigin` and related nodes, then when it's ready, set the new `XROrigin` to current, and then unload the old scene.